tests/test-cases/Group1-Docker-Commands/1-10-Docker-PS.robot

```diff
@@ -187,6 +187,15 @@ Docker ps ports output
     Should Not Contain  ${output}  8002->80/tcp
 
 
+Docker create output with specified static IP
+    ${rc}=  Run And Return Rc  docker %{VCH-PARAMS} pull ${nginx}
+    Should Be Equal As Integers  ${rc}  0
+
+    # published via the container-network with specified static ip
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} create -p 8080 --net=public --ip 60.0.9.150 ${nginx}
+    Should Contain  ${output}  Cannot set static IP (60.0.9.150)
```

Comment:
Checking for the full message will help make the test self documenting given it is actually a useful error message.

Reply:
Done! Add full message.